var/spack/repos/builtin/packages/openldap/package.py

```diff
@@ -17,8 +17,9 @@ class Openldap(AutotoolsPackage):
     """
 
     homepage = "https://www.openldap.org/"
-    url      = "ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-2.4.48.tgz"
+    url      = "http://gpl.savoirfairelinux.net/pub/mirrors/openldap/openldap-release/openldap-2.4.49.tgz"
```

Comment:
Why are we changing the `url`?

Reply:
It's just the same as #14835 , maybe some problem with the network in my side.
I restored the url in patch https://github.com/spack/spack/pull/14815/commits/dccc3bdf103646509e6778e4c47c24722eb4b429